app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ActiveRecord::Base
+  validates :image, presence: true
+
+  has_attached_file :image, styles: { :medium => "640x" }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)